name-service/js/package.json

```
@@ -23,25 +23,35 @@
     "prepublish": "tsc",
     "lint": "yarn pretty && eslint --max-warnings 0 'src/*.ts'",
     "lint:fix": "yarn pretty:fix && eslint 'src/*.ts' --fix",
-    "pretty": "prettier --check 'src/*.ts'",
-    "pretty:fix": "prettier --write 'src/*.ts'",
-    "doc": "yarn typedoc src/index.ts"
+    "pretty": "prettier --check '{src/*.ts,test/*/*.ts}'",
+    "pretty:fix": "prettier --write '{src/*.ts,test/*/*.ts}'",
+    "doc": "yarn typedoc src/index.ts",
+    "test": "npm run test:unit && npm run test:e2e",
```

nit for consistency:
```suggestion
    "test": "yarn test:unit && yarn test:e2e",
```